### DISCUSSION TITLE
怎么更新了就白版了啊

### DISCUSSION THREAD

**Discussion by vpn-web** [AlistGo/alist#6103] [Q&A]
怎么更新了就白版了啊

**Comment by Chimeng1314** [comment]
不更新也白版，不知道问题出在哪里了，用得好好的，也没更改配置啥的

**Comment by fzlychee2019** [comment]
和您一样，最近突然白板了，局域网访问一切正常，远程访问，都是一片白板

**Comment by xhofe** [comment]
https://github.com/alist-org/alist/issues/6100#issuecomment-1964071872

**Reply by Chimeng1314** [reply]
已解决，大佬们牛逼
